### ISSUE TITLE
Spotify cannot perform a search after a while

### ISSUE THREAD

**Issue by maboleth** [mrpond/BlockTheSpot#514]
Using Spotify 1.2.29.605 installed as a windows app from Spotify website.

Adblock always works, no problems with that.

But after a while or after a PC was put to sleep for a prolonged time, Spotify app cannot search for new artists or check any other info. It can still play songs on your list though (probably from cache).

Workaround: Restart Spotify

Expected behaviour: Spotify search and other features always working, regardless of the time passed in the app or Sleep.

**Comment by mrpond** [comment]
Hi, I can't reproduce this issue, maybe what is your windows version?

**Comment by Nuzair46** [comment]
This sounds like an issue with Spotify app itself and not a problem introduced by BlockTheSpot. Reopen if you think Im wrong.

**Comment by maboleth** [comment]
Well, the app behaved nicely and without any problems before patching/blockthespot.

Afterwards I started having these kind of issues with it. So I think this is related. I also had 30d free for S Premium and it worked as advertised with no issues.

**Comment by Nuzair46** [comment]
There is nothing in the adblocker that messes with search function.
You can try disabling the ad blocker and then see if you face this issue. As this issue is not observed with anyone else, it could be just Spotify or maybe issue with your specific installation.

**Comment by scrouthtv** [comment]
I am also having this issue. Currently on 
```
Spotify for Windows (64 bit)
1.2.31.1205.g4d59ad7c
xpui_2024-02-09_1707492707686_4d59ad7
cef_121.3.6+gbbcaed4+chromium-121.0.6167.139
Runtime: Alloy
```
on Windows 11 23H2 / Build 22631.3007

**Comment by maboleth** [comment]
![Untitled-1](https://github.com/mrpond/BlockTheSpot/assets/2467520/41d02385-d0d1-453f-85aa-26fb39583f32)

Here's how it looks like. Notice the unpopulated areas to the right. 
Those should be info about the band, song etc. Also search doesn't work.

**Comment by Nuzair46** [comment]
Does this only happen after sleep?
Did you guys try replicating this issue with clean Spotify installation?

**Comment by maboleth** [comment]
> Does this only happen after sleep? Did you guys try replicating this issue with clean Spotify installation?

Now that you said I did an extensive troubleshoot.

I've uninstalled it all. Also confirmed there was no Spotify leftovers in any of the dirs.

Then I've reinstalled Spotify from their website - using the specified standalone install.
After that I could see the window for AD placements but there was no ads. Also no song interrupts for commercials.

Almost if some ad-blocker was still in place (I used none except BlockTheSpot). And yes, the search/song info still disappeared after Sleep.

**Comment by scrouthtv** [comment]
Yeah, for me it's on clean install too.

I was encountering this issue for quite some weeks now, so I cleanly reinstalled the newest Spotify from their website. I'm still encountering this issue.

I also just checked, it instantly breaks when putting my pc to sleep. I can turn it back on instantly after it goes to sleep, and spotify is broken.

The search looks like this for me:
![image](https://github.com/mrpond/BlockTheSpot/assets/23227405/20aafaf6-802c-44d3-be5b-26131e0bf5e8)

**Comment by maboleth** [comment]
Well I can confirm now this happens without this adblock installed.

Just did the clean install Spotify recommended, with the shutdown sequence.
Same happens. I do have ads now, but after sleep Spotify is dead, except the songs that are/were already there.

**Comment by maboleth** [comment]
Apparently this is a (known?) bug in Spotify.
https://community.spotify.com/t5/Desktop-Windows/BUG-Spotify-Home-Page-no-loading/m-p/5798674

Sorry to cause trouble for you guys, but it was an unfortunate event where Spotify started behaving oddly just after I installed the adblocker (previously I used Spotify from the MS Store that had no problems). So blaming the blocker was the first thing that was suspicious.
